Dockerfile

```diff
@@ -64,13 +64,7 @@ ARG DEPLOYMENT=prod
 RUN echo "Deploying for ${DEPLOYMENT}"
 
 # Build the app.
-RUN npx cross-env REACT_APP_DEPLOYMENT=${DEPLOYMENT} npm run build
-
-# The step below is a JavaScript module that fixes a bug 
-# related to the paths of static assets in a React application. 
-# The bug is caused by a known issue in the create-react-app package, 
-# which results in duplicate static/js entries in the paths of some chunk files.
-RUN npm run fix-web-dev
```

Comment:
I hope that this issue won't be a problem again:
https://github.com/yaptide/ui/issues/640